internal/cloud/realm/app_test.go

```diff
@@ -34,13 +35,37 @@ func TestRealmApps(t *testing.T) {
 			assert.Equal(t, name, app.Name)
 			assert.True(t, strings.HasPrefix(app.ClientAppID, name), "expected client app id to be prefixed with name")
 
-			t.Run("And find the app by id", func(t *testing.T) {
+			t.Run("and find the app by id", func(t *testing.T) {
 				apps, err := client.FindApps(realm.AppFilter{App: app.ClientAppID})
 				assert.Nil(t, err)
 				assert.Equal(t, []realm.App{app}, apps)
 			})
 
-			t.Run("And delete the app by id", func(t *testing.T) {
+			t.Run("and get the app description by id", func(t *testing.T) {
+				appDesc, err := client.AppDescription(groupID, app.ID)
+				assert.Nil(t, err)
+				assert.Match(t, realm.AppDescription{
+					ClientAppID:    app.ClientAppID,
+					Name:           app.Name,
+					RealmURL:       fmt.Sprintf("%s/groups/%s/apps/%s/dashboard", u.RealmServerURL(), groupID, app.ID),
+					DataSources:    []realm.DataSourceSummary{},
+					HTTPEndpoints:  []realm.HTTPEndpointSummary{},
+					ServiceDescs:   []realm.ServiceSummary{},
+					AuthProviders:  []realm.AuthProviderSummary{{"api-key", "api-key", false}},
+					CustomUserData: realm.CustomUserDataSummary{},
+					Values:         []string{},
+					Functions:      []realm.FunctionSummary{},
+					Sync:           realm.SyncSummary{},
+					GraphQL: realm.GraphQLSummary{
+						URL:             fmt.Sprintf("%s/api/client/v2.0/app/%s/graphql", u.RealmServerURL(), app.ClientAppID),
```

Comment:
I have discovered the issue. If we want the server to run on port 9090 in the `test_cli_config.json` we need to add `"api": {"listenAddresses": [{"hostname": "0.0.0.0", "port": 9090}]` to the file. Currently, it is defaulting to port 8080, and since the `baseUrl`s appear to not affect the listening endpoints and therefore cause the difference. The reason I'm assuming we haven't crossed this issue until now is that the `.evg.yml` is targetting the defaulted port of 8080. So either way, the `test_cli_config.json` and the `.evg.yml` need to be updated to be consistent with the port we want to run against.

Reply:
let's update the `test_cli_config.json` with the `listenAddresses` like you suggest here, which then if I'm understanding properly, after we merge that we'll need to make the corresponding changes here in the `.evg.yml`

good chance I missed something here though, lmk if that doesn't work